henge.go

```diff
@@ -40,7 +41,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	err = transformers.Transform(*provider, flag.Args()[0:]...)
+	err = transformers.Transform(*provider, *interactive, flag.Args()[0:]...)
```

Comment:
I think this is ok for now. 
This is going to be better after we switch to cobra.